Pidaq/Lib/dts_manager/dts_manager.py

```diff
@@ -0,0 +1,409 @@
+
+        def get_current_data(self, start=0, stop=8):
+            return self.bus.messages[self.current_info_id].data[start:stop]
+
+        def get_voltage_data(self, start=0, stop=8):
+            return self.bus.messages[self.voltage_info_id].data[start:stop]
+
+        def get_flux_data(self, start=0, stop=8):
+            return self.bus.messages[self.flux_info_id].data[start:stop]
+
+        def get_internal_voltages_data(self, start=0, stop=8):
+            return self.bus.messages[self.internal_voltages_id].data[start:stop]
+
+        def get_internal_states_data(self, start=0, stop=8):
+            return self.bus.messages[self.internal_states_id].data[start:stop]
+
+        def get_faults_data(self, start=0, stop=8):
+            return self.bus.messages[self.fault_codes_id].data[start:stop]
+
+        def get_torque_timer_data(self, start=0, stop=8):
+            return self.bus.messages[self.torque_timer_id].data[start:stop]
+
+        def get_modulation_index_data(self, start=0, stop=8):
+            return self.bus.messages[self.modulation_index_id].data[start:stop]
+
+        def convert_temperatures(self) -> None:
+            if self.get_temp1_data():
+                self.module_a_temperature = int.from_bytes(
+                    self.get_temp1_data(0, 2), byteorder='little', signed=True) / 10
+                self.module_b_temperature = int.from_bytes(
+                    self.get_temp1_data(2, 4), byteorder='little', signed=True) / 10
+                self.module_c_temperature = int.from_bytes(
+                    self.get_temp1_data(4, 6), byteorder='little', signed=True) / 10
+                self.gate_driver_board_temperature = int.from_bytes(
+                    self.get_temp1_data(6, 8), byteorder='little', signed=True) / 10
+            if self.get_temp2_data():
+                self.control_board_temperature = int.from_bytes(
+                    self.get_temp2_data(0, 2), byteorder='little', signed=True) / 10
+                self.rtd_1_temperature = int.from_bytes(
+                    self.get_temp2_data(2, 4), byteorder='little', signed=True) / 10
+                self.rtd_2_temperature = int.from_bytes(
+                    self.get_temp2_data(4, 6), byteorder='little', signed=True) / 10
+                self.rtd_3_temperature = int.from_bytes(
+                    self.get_temp2_data(6, 8), byteorder='little', signed=True) / 10
+            if self.get_temp3_data():
+                self.rtd_4_temperature = int.from_bytes(
+                    self.get_temp3_data(0, 2), byteorder='little', signed=True) / 10
+                self.rtd_5_temperature = int.from_bytes(
+                    self.get_temp3_data(2, 4), byteorder='little', signed=True) / 10
+                self.motor_temperature = int.from_bytes(
+                    self.get_temp3_data(4, 6), byteorder='little', signed=True) / 10
+
+        def convert_low_voltages(self) -> None:
```

Comment:
Not sure if convert is the ideal name for these methods. They do convert the data from bytes to integers, but they also read the bus and update the class field. How do you feel about using update for these methods?

Reply:
Yeah, that probably describes them a bit better